lib/User/OfflineUser.php

```diff
@@ -227,4 +227,17 @@ protected function determineShares() {
 
 		$this->hasActiveShares = false;
 	}
+	
+	/**
+	 * Adding empty function to OfflineUser to avoid throwing error "function missing".
+	 * Composes the display name and stores it in the database. The final
+	 * display name is returned.
```

Comment:
I think the description should only say what the function does, so: "Returns the given display name and doesn't store anything as this user is deleted"